.github/workflows/ci-build.yaml

```diff
@@ -96,22 +98,23 @@ jobs:
             install_k3s_version: v1.25.11-k3s1
             profile: minimal
     steps:
-      - name: Install socat
-        # needed by Kubernetes v1.25
+      - name: Install socat (needed by Kubernetes v1.25)
         run: sudo apt-get -y install socat
       - uses: actions/checkout@v3
       - uses: actions/setup-go@v4
         with:
           go-version: "1.21"
           cache: true
-      - uses: actions/setup-java@v3
+      - name: Install Java for the SDK
         if: ${{matrix.test == 'test-java-sdk'}}
+        uses: actions/setup-java@v3
         with:
           java-version: '8'
           distribution: adopt
           cache: maven
-      - uses: actions/setup-python@v4
+      - name: Install Python for the SDK
```

Comment:
Isn't it already intuitive by using this action?

Reply:
I did not know why Python was being installed, given that it's not a language used inside of Argo. The SDK has a test here, and I mention the SDK in the name